omod/src/main/java/org/openmrs/module/htmlformentryui/page/controller/htmlform/ViewEncounterWithHtmlFormPageController.java

```diff
@@ -52,8 +65,13 @@ public void get(@RequestParam("encounter") Encounter encounter,
         model.addAttribute("showPatientHeader", showPatientHeader);
 
         HtmlForm htmlForm = htmlFormEntryService.getHtmlFormByForm(encounter.getForm());
+        
         if (htmlForm == null) {
-            throw new IllegalArgumentException("encounter.form is not an HTML Form: " + encounter.getForm());
+             message = messageSourceService.getMessage("encounter.form is not an HTML Form" +encounter.getForm());
```

Comment:
i gues this should take in a message code ,

Reply:
It depends, it takes in both Actually message codes would be also another good idea since we are trying to hide the exception hence returning a readable message format